codice/ddf
# DDF-5703 Extract core features from apps

#### What does this PR do?
Extracts the core catalog features from `-app` features.  Creates a new `ddf-catalog` distribution based on these features that is 300mb in size.

#### Who is reviewing it? 


#### Select relevant component teams: 


#### Ask 2 committers to review/merge the PR and tag them here.

@coyotesqrl
@rzwiefel

#### How should this be tested?


- Verify DDF distribution still works using the standard profile
- Verify `distribution/ddf-catalog` distribution by running `feature:install profile-catalog` at the command line
- Ingest data using command line and/or REST endpoint
- Query using command line and/or OpenSearch endpoint

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #5703

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
